src/partitionmanager_p.h

```diff
@@ -102,6 +102,9 @@ class PartitionManagerPrivate : public QObject, public QSharedData
 
     // Allow direct access to the Partitions.
     friend class UDisks2::Monitor;
+
+private slots:
+    void onStatusChanged(const QString &devicePath, Partition::Status status);
```

Comment:
In my opinion, not a very good name for the slot

Reply:
Why do you think so? This slot is for signal status(). What name is better for it?

Reply:
`statusChanged` perhaps better to perception

Reply:
It is a better name for signal actually. Slot is a reaction. That is why "on". "onStatus" isn't a correct name. Thats why i used "onStatusChanged"

Reply:
Done